apps/cms/config/sync/user.role.administrator.yml

```diff
@@ -42,6 +44,7 @@ permissions:
   - 'dropzone upload files'
   - 'edit config_pages entity'
   - 'translate editable entities'
+  - 'trigger a gatsby build'
```

Comment:
Added `trigger a gatsby build` and `access publisher` permissions to the administrator role as these ones are going in pairs (if we access publisher, we can trigger a build). But we need to check if it's fine for this role.